worksheet_A/TerminalHacking.cpp

```diff
@@ -41,6 +45,66 @@ int main()
 
 	// TODO: implement the rest of the game
 
+	// initialise variables
+	bool WordInList = false;
+	std::cout << secret;
+	std::string guess;
+	int lives = 4;
+	std::string result = "Lost";
```

Comment:
Do all of these need to be declared outside the loop? Generally good to keep variable declarations close to the point of use.